README.md

```diff
@@ -1,2 +1,59 @@
+
+The purpose of this documentary is to document the configuration and plugins of each ArchivesSpace environement we use:
+- `development`: refers to the local development environment each developer is running on their machine.
+- `test`: refers to the remote test instance currently managed by Lyrasis. This is the instance where all testing is done.
+- `production`: refers to the ArchivesSpace production instance, which is also managed by Lyrasis.
+
+Each directory in this repository is named after one of the environments listed above. Under each directory, there are two files:
+
+- `config.rb`: contains all the custom configurations to the ArchivesSpace environment
+- `plugins.yml`: contains a list of all the plugins included in the the ArchivesSpace environment. Each plugin must have the following attributes:
+
+```yml
+plugins:
+  - name: aspace_static_plugins # matches git repository name
+    git: true
+    branch: "v0.2.1" # matches the version tag or branch in the repository
+    url: https://github.com/emory-libraries/aspace_static_plugins.git # url of the plugin's git repository
+```
+
+### Plugins
+
+At Emory Libraries, we manage all ArchivesSpace UI customizations under one plugin, [aspace_static_plugins](https://github.com/emory-libraries/aspace_static_plugins). We also use other plugins listed under `plugins.yml` of each environment e.g. [container_management_labels](https://github.com/emory-libraries/container_management_labels).
+
+### Deployment Process
+
+1. Create a new release for [aspace_static_plugins](https://github.com/emory-libraries/aspace_static_plugins) if the latest release does not contain the most up to date changes, and ensure the new release has a tag number that follows semantic versioning. You can use the following [instructions](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository) to create a new release.
+2. Confirm which environment you are deploying to e.g. test, production
+3. Verify the `plugins.yml` of the environment you are deploying to. Insure all plugins required are listed, with a correct version/branch and repository url.
+4. Verify the `config.rb` of the environment you are deploying to. Ensure all configuration customizations are included, and ensure `AppConfig[:emory_plugins_version]` is set to the version of the [aspace_static_plugins](https://github.com/emory-libraries/aspace_static_plugins) plugin you are deploying (listed under `plugins.yml`).
+5. Create a new release of this repository and choose a tag that follows semantic versioning. You will use this new release to complete the following two tasks.
+6. Download the `config.rb` that matches the release tag above and environment you are deploying.
+7. Download the `plugins.yml` that matches the release tag and environment you are deploying.
+8. Reach out to the Lyrasis team by email, using the following template. Make sure you attach the `config.rb` and `plugins.yml` files downloaded above.
```

Comment:
@abelemlih and @rotated8 Should we list the email address here, or avoid that?

Reply:
I think it's probably better to omit it.